bundles/org.openhab.binding.nibeuplink/src/main/java/org/openhab/binding/nibeuplink/internal/config/NibeUplinkConfiguration.java

```diff
@@ -91,9 +90,9 @@ public void setHouseKeepingInterval(Integer houseKeepingInterval) {
 
     @Override
     public String toString() {
-        return new ToStringBuilder(this).append("user", getUser()).append("password", getPassword())
-                .append("nibeId", getNibeId()).append("pollingInterval", getPollingInterval())
-                .append("houseKeepingInterval", getHouseKeepingInterval()).append("asyncTimeout", getAsyncTimeout())
-                .append("syncTimeout", getSyncTimeout()).toString();
+        return getClass().getSimpleName() + "{ user=" + getUser() + ", password=" + getPassword() + ", nibeId='"
+                + getNibeId() + ", pollingInterval=" + getPollingInterval() + ", houseKeepingInterval="
+                + getHouseKeepingInterval() + ", asyncTimeout=" + getAsyncTimeout() + ", syncTimeout="
+                + getSyncTimeout().toString() + "}";
```

Comment:
No need of the `.toString()` after `getSyncTimeout()`

Reply:
yes